firebase-ml-modeldownloader/firebase-ml-modeldownloader.gradle

```diff
@@ -43,9 +43,15 @@ android {
 }
 
 dependencies {
+    implementation project(':encoders:firebase-encoders')
+    implementation project(':encoders:firebase-encoders-json')
     implementation project(':firebase-common')
     implementation project(':firebase-components')
+    implementation project(':firebase-datatransport')
     implementation project(':firebase-installations-interop')
+    implementation project(':transport:transport-api')
+    implementation project(':transport:transport-backend-cct')
+    implementation project(':transport:transport-runtime')
```

Comment:
There is no need to declare these deps, please only depend on `transport-api` and `firebase-datatransport`

Reply:
Done